CorePayments/src/test/java/com/paypal/android/corepayments/api/CoreRequestParserUnitTest.kt

```diff
@@ -0,0 +1,177 @@
+package com.paypal.android.corepayments.api
+
+import com.paypal.android.corepayments.Code
+import com.paypal.android.corepayments.HttpResponse
+import com.paypal.android.corepayments.PayPalSDKError
+import com.paypal.android.corepayments.PaymentsJSON
+import com.paypal.android.corepayments.api.models.GetOrderInfoResponse
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Assert
+import org.junit.Test
+import org.junit.experimental.runners.Enclosed
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import java.net.HttpURLConnection
+
+@RunWith(Enclosed::class)
+class CoreRequestParserUnitTest {
+
+    class CoreRequestParserFunctionsUnitTest {
+
+        private val correlationID = "correlationID"
+        private val headers: Map<String, String> = mapOf(
+            "Paypal-Debug-Id" to correlationID,
+        )
+
+        @Test
+        fun `it parses a get info order response`() {
+            val response = """ 
+            {
+                "id":"89H32494DY5382259",
+                "intent":"AUTHORIZE",
+                "status":"CREATED",
+                "payment_source":{
+                    "card":{
+                        "last_digits":"2661",
+                        "brand":"VISA",
+                        "type":"CREDIT",
+                        "authentication_result":{
+                            "liability_shift":"NO",
+                            "three_d_secure":{
+                                "enrollment_status":"U"
+                            }
+                        }
+                    }
+                },
+                "purchase_units":[
```

Comment:
@jcnoriega - Are we OK to the 1st value in this array as the amount to act on? What if the array contains multiple amounts.